.lintstagedrc

```diff
@@ -1,3 +1,6 @@
 {
-    "*.php": "phplint"
+    "*.php": [
+        "phplint",
+        "vendor/bin/phpcs"
```

Comment:
`lintstaged` will append the list of the modified files to the command; this will speed things up compared to eslines (with eslines, phpcs had to scan the entire codebase, and then eslines filtered out all lines not related to the current PR).